nodes/cma/nodes-cma-autoscaling-custom-trigger-auth.adoc

```diff
@@ -47,20 +61,74 @@ metadata: <1>
   name: secret-cluster-triggerauthentication
 spec:
   secretTargetRef: <2>
-  - parameter: user-name <3>
-    name: secret-name <4>
-    key: USER_NAME <5>
-  - parameter: user-password
+  - parameter: username <3>
+    name: my-secret <4>
+    key: username <5>
+  - parameter: password
     name: secret-name
-    key: USER_PASSWORD
+    key: password
 ----
 <1> Note that no namespace is used with a cluster trigger authentication.
-<2> Specifies that this trigger authentication uses a secret for authorization.
+<2> Specifies that this trigger authentication uses a secret for authorization when connecting to the metrics endpoint.
 <3> Specifies the authentication parameter to supply by using the secret.
 <4> Specifies the name of the secret to use.
 <5> Specifies the key in the secret to use with the specified parameter.
 
-.Example trigger authentication with a token
+.Example secret with certificate authority (CA) details
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: my-secret
+  namespace: my-namespace
+data: 
+  ca-cert.pem: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0... <1>
+  client-cert.pem: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0...
+  client-key.pem: LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0t...
+----
+<1> Specifies the TLS certificates and key for TLS client authentication. The values  in a `data` stanza must be base-64 encoded.
```

Comment:
I think we should probably have a separate call-out for `ca-cert.pem` which says something like "Specifies the TLS CA Certificate for authentication of the metrics endpoint." Then the current call-out should be just for `client-cert.pem` and `client-key.pem`.